staging/vhost-device-sound/src/stream.rs

```diff
@@ -276,13 +283,46 @@ impl Buffer {
 
         Ok(())
     }
+
+    pub fn prepare_write(&mut self) {
```

Comment:
maybe we should have the names matching read() and write() or prepare_read() and prepare_write() WDYT

Reply:
Sounds good. `prepare_read`, `prepare_write` or perhaps `prepare_playback`, `prepare_capture`?

Reply:
Changed